### DISCUSSION TITLE
Support for VStream

### DISCUSSION THREAD

**Discussion by reinhardtlr** [3dproger/AxelChat#342] [General]
Hello! 

I am wondering if you could add support for VStream? It is a new streaming platform catered for VTuber. They just recently released API for their website so I don't know if that is all that's needed to add support. Thank you for your time in reading and hoping to wait for good news!

**Comment by 3dproger** [comment]
Hello!

Most likely this is possible and I will try to do this in future updates.

Thanks for the offer!

**Comment by 3dproger** [comment]
Apparently this discussion is no longer relevant as the VStream is no longer running: https://vstream.com/adieu

**Comment by 3dproger** [comment]
Hi!

I think the discussion can be closed. If there are questions again, you can reopen the discussion or create a new one
